### ISSUE TITLE
Unable to access 4chan.org

### ISSUE THREAD

**Issue by iloveyumyumshrimpnoodles** [OtterBrowser/otter-browser#1750]
# Description
Visiting 4chan.org results in an infinite loop.
Perhaps it's not 4chan, but cloudflare?

# Debug info
## Version
Otter 1.0.02 
Web browser controlled by the user, not vice-versa.
https://www.otter-browser.org/

Web backend: WebKit Backend 602.1.

SSL library version: OpenSSL 1.1.1t 7 Feb 2023.

## System info
Uname: `Linux host 6.1.31_1 #1 SMP PREEMPT_DYNAMIC Wed May 31 05:53:37 UTC 2023 x86_64 GNU/Linux`
OS: `Void Linux x86_64 `
Host: `VivoBook_ASUSLaptop X515UA_M515UA 1.0`

# Images
![Screenshot_2023-06-13_20-30-19](https://github.com/OtterBrowser/otter-browser/assets/125209101/6fa1a220-28de-4793-8c89-0657f7b3624d)
![Screenshot_2023-06-13_20-30-44](https://github.com/OtterBrowser/otter-browser/assets/125209101/bf1acaf6-4da1-4e82-941f-a5aa30d758ac)

**Comment by iloveyumyumshrimpnoodles** [comment]
Yeah, it seems to be cloudflare related. https://dash.cloudflare.com/login?lang=en-US also doesn't play nice:
![image](https://github.com/OtterBrowser/otter-browser/assets/125209101/33cdabe4-c08b-4737-959f-bab962c313de)

**Comment by iloveyumyumshrimpnoodles** [comment]
just noticed i forgot to change the name 2 weeks ago, my bad

**Comment by beastie1** [comment]
But then, why does it work in both Chromium and Falkon?
